### DISCUSSION TITLE
셀피카메라 모듈 에러 현상

### DISCUSSION THREAD

**Discussion by jgw1413** [naverz/zepeto-studio-kor#1173] [5. Q&A]
안녕하세요
기존 제페토 프로젝트에 셀피카메라 모듈 추가하려고 합니다.
근데 씬에 모듈 배치하고 셀피 버튼을 누르면 SelfiCamera.ts 에서 에러가 나옵니다.
에러 내용은 아래와 같습니다.
TypeError: Cannot read property 'InitSetting' of null
    at ScreenShotModeManager.StartScreenShotMode

분명 SelfiCamera 프리팹 생성고 스트립트도 모두 존재하는 것 확인했습니다...
기존 샘플에서 바꾼게 없어요 ㅜㅜ

zepeto.world 버전은 1.5.1입니다
제페토 스튜디오에서 제공하는 멀티샘플 예제 사용하였습니다

**Comment by zlmrzlo** [comment]
인스펙터에 들어갈 변수들이 빠져있는지 먼저 확인을 해보는 것이 좋을 것 같아요.

**Comment by ZdevMonZ** [comment]
메타파일이나 인풋매니저 쪽 보시면 좋을 것 같아요.
https://github.com/naverz/zepeto-studio-kor/discussions/1124
https://github.com/naverz/zepeto-studio-kor/discussions/581#discussioncomment-2506605

**Reply by jgw1413** [reply]
모두 정상적으로 체크 되어있었고 메타파일도 삭제 해보았습니다.
그래도 안되다가 분명 셀피카메라 프리팹이 생성되었고 왜 InitSetting를 참고하지 못하는지 디버깅 해보니 갑자기 정상적으로 참조 되었습니다...
아니면 메타파일 삭제 후 정상처리 되는게 늦었던 건지도 모르겠습니다
참조가 잘 안되시는 분은 메타파일삭제나 디버깅 해보시길 바랍니다.
아무튼 정상적으로 잘 해결됐습니다. 감사합니다!
